irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -76,8 +134,31 @@ namespace iroha {
 
       log_->info("transactions in verified proposal: {}",
                  validated_proposal.transactions().size());
-      return std::make_shared<decltype(validated_proposal)>(
-          validated_proposal.getTransport());
+      return std::make_pair(std::make_shared<decltype(validated_proposal)>(
+                                validated_proposal.getTransport()),
+                            transactions_errors_log);
+    }  // namespace validation
```

Comment:
`// namespace validation` does not look right, it should be removed because namespace is closed below.